frontend/catalyst/compilation_pipelines.py

```diff
@@ -777,9 +777,10 @@ def compute_jvp(self, primals, tangents):
             tangent = tangents[arg_idx]  # [1]
             taxis = list(range(tangent.ndim))
             for res_idx in range(len(results_data)):
-                deriv_idx = diff_arg_idx * len(results_data) + res_idx
+                deriv_idx = diff_arg_idx + res_idx * len(argnums)
```

Comment:
Could you please check the comment in the header of this function? Should we adjust shapes in the example there as well?

Reply:
Yes I will update that, good catch :+1: